DQM/L1TMonitorClient/python/L1TStage2MonitorClient_cff.py

```diff
@@ -16,38 +16,37 @@
 # DQM quality tests 
 from DQM.L1TMonitorClient.L1TStage2QualityTests_cff import *
 
+# L1T Objects Ration Timing Plots
+from DQM.L1TMonitorClient.L1TObjectsTimingClient_cff import *
+
 # L1 event info DQM client 
 from DQM.L1TMonitorClient.L1TStage2EventInfoClient_cfi import *
 
 # CaloLayer2 client
 from DQM.L1TMonitorClient.L1TStage2CaloLayer2Client_cff import *
 
-# BMTF client
-from DQM.L1TMonitorClient.L1TStage2BMTFClient_cff import *
-
 # uGMT client
 from DQM.L1TMonitorClient.L1TStage2uGMTClient_cff import *
 
 # uGT client
 from DQM.L1TMonitorClient.L1TStage2uGTClient_cff import *
 
-# L1 event info DQM client EMTF 
-from DQM.L1TMonitorClient.L1TStage2EMTFEventInfoClient_cfi import *
-
+#
 # define sequences 
 #
 
 # L1T monitor client sequence (system clients and quality tests)
 l1TStage2Clients = cms.Sequence(
                         l1tStage2EventInfoClient
                       + l1tStage2uGTCaloLayer2CompClient
-                      + l1tStage2BmtfClient
                       + l1tStage2uGMTClient
                       + l1tStage2uGTClient
-                      + l1tStage2EMTFEventInfoClient
+                      + l1tObjectsTimingClient
                         )
 
 l1tStage2MonitorClient = cms.Sequence(
-                        l1TStage2QualityTests 
-                      + l1TStage2Clients 
+                        l1TStage2QualityTests +
+                        l1TStage2Clients 
+                       + l1tObjectsTimingClient
```

Comment:
Since l1tObjectsTimingClient is aready in the l1TStage2Clients sequence this sequence here does not need to be changed.